packages/shared/src/graphql/actions.ts

```diff
@@ -26,6 +26,7 @@ export enum ActionType {
   HidePublicSquadEligibilityCard = 'hide_public_squad_eligibility_card',
   HidePublicSquadStep = 'hide_public_squad_step',
   MakeSquadPublic = 'make_squad_public',
+  HidePublicSquadAbout = 'hide_public_squad_about',
```

Comment:
Now this does not hide the step rather it marks it completed right?

Reply:
Actually is marked as done, I'm changing the name to reflect the behaviour! Thanks ✅